build/ci/Makefile

```diff
@@ -93,6 +93,7 @@ ci-release: vault-public-key vault-docker-creds
     	-e "ELASTIC_DOCKER_PASSWORD=$(shell cat $(DOCKER_CREDENTIALS_FILE))" \
     	-e "USE_ELASTIC_DOCKER_REGISTRY=true" \
     	-e "OPERATOR_IMAGE=$(OPERATOR_IMAGE)" \
+    	-e "LATEST_RELEASED_IMG=$(LATEST_RELEASED_IMG)" \
```

Comment:
Where are we setting `LATEST_RELEASED_IMG`?

Reply:
Because of https://github.com/elastic/cloud-on-k8s/blob/master/operators/Makefile#L170

Reply:
It's set in Jenkins pipeline here: https://github.com/elastic/cloud-on-k8s/blob/master/build/ci/release/Jenkinsfile#L19